src/main.cpp

```diff
@@ -62,11 +64,11 @@ void setup()
 void setGainsFromRcTrimmers()
 {
   controller.setQuaternionGain(
-      -radio.trimmerVRAPercentage() * RADIO_TRIMMER_MAX_QUATERNION_XY_GAIN);
+      - 0.470 * RADIO_TRIMMER_MAX_QUATERNION_XY_GAIN);
```

Comment:
These values are now fixed. Why don't you define them as new constants. Constants like `RADIO_TRIMMER_MAX_QUATERNION_XY_GAIN` are not needed any more.

Reply:
Done! Now the constant gains are defined as `QUATERNION_XY_GAIN`, `OMEGA_XY_GAIN`, and `OMEGA_Z_GAIN`.